src/qml/AboutPage.qml

```diff
@@ -88,6 +88,8 @@ Flickable {
                 { label: qsTr("WLAN MAC"), text: about.wlanMacAddress },
                 { label: qsTr("IMEI"), text: about.imei },
                 { label: qsTr("Serial number"), text: about.serial },
+                { label: qsTr("Vendor"), text: about.vendorName },
```

Comment:
Maybe "Manufacturer" and "Model" would be clearer and more concise? Not sure about that, let's ask a native @beroset

Reply:
Maybe I'm just being pedantic, but "Fossil" doesn't actually manufacture the devices with their name on them -- it's some contract manufacturing organization.  That's my logic for calling it "Vendor," especially since some watches are sold under multiple "Vendor" names even though they're really manufactured in the same factory (or at least the boards are).  Example: `ray` is sold as Misfit Vapor, Emporio Armani, Fossiq Q, Kate Spade, Michael Kors, Skagen Falster, and Tory Burch.

Reply:
Thanks! Vendor always sounded weird to me but I'm not a native :) so let's keep vendor and use the shorter "model" then

Reply:
Changed to Model and Vendor